deployment/macos/mobiledevicemanagement/mobile_config_gen.py

```diff
@@ -0,0 +1,228 @@
+            <string>{2}</string>
+            <key>PayloadVersion</key>
+            <integer>1</integer>
+        </dict>
+        <dict>
+            <key>NotificationSettings</key>
+            <array>
+                <dict>
+                    <key>AlertType</key>
+                    <integer>2</integer>
+                    <key>BadgesEnabled</key>
+                    <true/>
+                    <key>BundleIdentifier</key>
+                    <string>co.elastic.alert</string>
+                    <key>CriticalAlertEnabled</key>
+                    <true/>
+                    <key>NotificationsEnabled</key>
+                    <true/>
+                    <key>ShowInLockScreen</key>
+                    <true/>
+                    <key>ShowInNotificationCenter</key>
+                    <true/>
+                    <key>SoundsEnabled</key>
+                    <true/>
+                </dict>
+            </array>
+            <key>PayloadDisplayName</key>
+            <string>Notifications Payload</string>
+            <key>PayloadIdentifier</key>
+            <string>com.apple.notificationsettings.{3}</string>
+            <key>PayloadOrganization</key>
+            <string>{6}</string>
+            <key>PayloadType</key>
+            <string>com.apple.notificationsettings</string>
+            <key>PayloadUUID</key>
+            <string>{3}</string>
+            <key>PayloadVersion</key>
+            <integer>1</integer>
+        </dict>
+    </array>
+    <key>PayloadDescription</key>
+    <string>Grants Elastic Agent the necessary permissions to secure your Mac</string>
+    <key>PayloadDisplayName</key>
+    <string>Elastic Agent Endpoint Configuration</string>
+    <key>PayloadEnabled</key>
+    <true/>
+    <key>PayloadIdentifier</key>
+    <string>{4}</string>
+    <key>PayloadOrganization</key>
+    <string>{6}</string>
+    <key>PayloadRemovalDisallowed</key>
+    <true/>
+    <key>PayloadScope</key>
+    <string>System</string>
+    <key>PayloadType</key>
+    <string>Configuration</string>
+    <key>PayloadUUID</key>
+    <string>{5}</string>
+    <key>PayloadVersion</key>
+    <integer>1</integer>
+</dict>
+</plist>
+
+"""
+
+def main(argv):
+
+    name = str()
+    output_file = str()
+
+    parser = argparse.ArgumentParser()
+    parser.add_argument("-n", "--name", help="The name of your company", action="store", required=True, type=str, dest="name")
+    parser.add_argument("-o", "--output", help="The absolute path to the mobileconfig that will be written out by this script", action="store", required=True, type=str, dest="output_file_path")
+
+    args = parser.parse_args()
+
+    output_file = args.output_file_path
+
+    # Ensure the file ends with .mobileconfig extension
+    if output_file.endswith(".mobileconfig") == False:
+        output_file += ".mobileconfig"
+
+
+    with open(output_file, 'w', encoding='utf-8') as output_config_file:
+
+        pos_args = [str(uuid.uuid4()).upper() for _ in range (0,6)]
```

Comment:
isn't `range(0,6)` the same as `range(6)`?

Reply:
effectively the same, I can update to simplify, thanks!